experimental/packages/opentelemetry-sdk-metrics-base/src/export/PeriodicExportingMetricReader.ts

```diff
@@ -63,7 +63,13 @@ export class PeriodicExportingMetricReader extends MetricReader {
 
   private async _runOnce(): Promise<void> {
     const metrics = await this.collect({});
+
     return new Promise((resolve, reject) => {
+      if (metrics === undefined) {
+        resolve();
```

Comment:
Why not return directly before the new promise so you don't have to do the resolve/return dance?

Reply:
Fixed it, thanks!